sdk/src/main/java/dev/cdevents/events/CustomTypeEvent.java

```diff
@@ -0,0 +1,192 @@
+// Code generated by dev.cdevents.generator.CDEventsGenerator. DO NOT EDIT.
+
+/*
+Copyright 2023 The CDEvents Authors
```

Comment:
2024?

Reply:
Actually it is generated from a common template for all the existing and new events.